test/io-tests/test_io.py

```diff
@@ -207,7 +207,7 @@ def authheader(token):
 
 def jwtauthheader(claim, secret):
     "Authorization header with signed JWT."
-    return authheader(jwt.encode(claim, secret).decode("utf-8"))
+    return authheader(jwt.encode(claim, secret))
```

Comment:
Is this change related to the other changes?

Reply:
Yes, the API of the Python jwt module changed with the newer version in the updated nixpkgs, returning a string instead of bytes: https://pyjwt.readthedocs.io/en/latest/changelog.html#jwt-encode-return-type